Prepare migration guide v5.2.0 → v5.2.1 (#1522)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/content/docs/5.2/migration.md

### Components

- **Back to top**
  - <span class="badge bg-danger">Breaking</span> Back to top 'Label inside' variant was removed because not compliant with Orange Design System. Even if the rendering could still work, it is recommended to only use the versions presented in the documentation.

- **Breadcrumb**
  - <span class="badge bg-success">New</span> Breadcrumb now has a dark variant.

- **Footers**
  - <span class="badge bg-warning">Warning</span> All Footers examples have been modified to use the "Terms and conditions" wording instead of "Terms & Conditions". Please reflect this modification into your websites.

- **List group**
  - <span class="badge bg-success">New</span> List group now has a dark variant.

- **Pagination**
  - <span class="badge bg-success">New</span> Pagination now has a dark variant.

- **Scrollspy**
  - <span class="badge bg-success">New</span> Scrollspy has a new `data-bs-threshold` data attribute.

- **Stepped process**
  - <span class="badge bg-success">New</span> Stepped process now has a dark variant.

- **Tags**

>   - <span class="badge bg-success">New</span> Tags now have a dark variant.

Reviewer: ```suggestion
  - <span class="badge bg-success">New</span> Tags are now a component.
```
Or something like this ?

- **Toasts**
  - <span class="badge bg-warning">Warning</span> Changed the markup for Toasts with a custom content (toast message and close button). Please reflect this modification into your websites.
    <details>
      <summary>More info</summary>
      {{< markdown >}}
```diff
<div class="toast align-items-center" role="alert" aria-live="assertive" aria-atomic="true">
  <div class="d-flex">

> -    <div class="toast-body">

Reviewer: ```suggestion
-   <div class="toast-body">
```
Same for all diff to have a proper alignment?

+    <div class="toast-body my-auto">
      Hello, world! This is a toast message.
    </div>
-    <button type="button" class="btn-close me-2 m-auto" data-bs-dismiss="toast"><span class="visually-hidden">Close</span></button>
+    <button type="button" class="btn-close ms-auto" data-bs-dismiss="toast"><span class="visually-hidden">Close</span></button>
  </div>
</div>
```
      {{< /markdown >}}
    </details>

- **Tooltips**

>   - <span class="badge bg-warning">Warning</span> Tooltips examples applied on SVGs have been updated to use `focusable="false"` and `aria-hidden="true"` because SVGs do not carry any specific information so they should not be readable by screen readers. Please reflect this modification in your websites.

Reviewer: Re reading this, I wonder if this change should be in the Tooltip section or inside the contents section without the mention of tooltips ?

- <span class="badge bg-warning">Warning</span> Changed the rendering of `<mark>` (and so `.mark`). Depending on your usage it may worth checking the impact in your websites.

- <span class="badge bg-warning">Warning</span> Tooltips examples applied on SVGs have been updated to use `focusable="false"` and `aria-hidden="true"` because SVGs do not carry any specific information so they should not be readable by screen readers. Please reflect this modification in your websites.
  <details class="mb-3">
    <summary>More info</summary>
    {{< markdown >}}
```diff
<a href="#" class="d-inline-block" data-bs-toggle="tooltip" data-bs-title="Default tooltip" aria-label="Default tooltip">
- <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 100 100">
+ <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 100 100" focusable="false" aria-hidden="true">
    <rect width="100%" height="100%" fill="#563d7c"/>
    <circle cx="50" cy="50" r="30" fill="#007bff"/>
  </svg>
</a>
```
    {{< /markdown >}}
  </details>

>   By the way check all the SVGs in your websites in order to apply this same modification if needed.

Reviewer: This line seems a bit lost IMO

      <li><code>$pagination-dark-active-item-color</code></li>
      <li><code>$pagination-dark-bg</code></li>
      <li><code>$pagination-dark-border-color</code></li>
      <li><code>$pagination-dark-color</code></li>
      <li><code>$pagination-dark-disabled-bg</code></li>
      <li><code>$pagination-dark-disabled-border-color</code></li>
      <li><code>$pagination-dark-disabled-color</code></li>
      <li><code>$pagination-dark-focus-bg</code></li>
      <li><code>$pagination-dark-focus-color</code></li>
      <li><code>$pagination-dark-hover-bg</code></li>
      <li><code>$pagination-dark-hover-border-color</code></li>
      <li><code>$pagination-dark-hover-color</code></li>
      <li><code>$step-item-dark-active-bg</code></li>
      <li><code>$step-item-dark-bg</code></li>
      <li><code>$step-item-dark-drop-shadow</code></li>
      <li><code>$step-item-dark-next-bg</code></li>
      <li><code>$step-link-dark-active-color</code></li>
      <li><code>$step-link-dark-color</code></li>
      <li><code>$step-link-dark-next-color</code></li>
      <li><code>$step-link-next-color</code></li>
    </ul>
  </details>

> - Dark text variants handling is now explained in [Customize > CSS variables > Dark text rule](https://boosted.orange.com/docs/5.2/customize/css-variables/#dark-text-rule).

Reviewer: Do we keep the link ? If yes, should be:
`{{< docsref "/customize/css-variables#dark-text-rule" >}}`

## Forms

> <!--- Boosted mod: no floating labels -->

Reviewer: To remove
